src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -399,12 +399,12 @@ internal Uri UriFromString(string uriString, UriKind uriKind, string localName,
             return UriFromString(UriParser, uriString, uriKind, localName, namespaceURI, reader);
         }
 
-        internal static Uri UriFromString(TryParseUri uriParser, string uriString, UriKind uriKind, string localName, string namespaceURI, XmlReader reader)
+        internal static Uri UriFromString(TryParseUriCallback uriParser, string uriString, UriKind uriKind, string localName, string namespaceURI, XmlReader reader)
         {
             Uri uri = null;
             var elementQualifiedName = new XmlQualifiedName(localName, namespaceURI);
-            var parseUriArgs = new ParseUriArgs(uriString, uriKind, elementQualifiedName);
-            object[] args = new object[] { parseUriArgs, uri };
+            var XmlUriData = new XmlUriData(uriString, uriKind, elementQualifiedName);
```

Comment:
Might be **x**mlUriData for local variable name to prevent confusion? There is another below.

Reply:
This is what happens when using case insensitive find/replace. Good catch, I'll fix it.